CHANGELOG.md

```diff
@@ -4,6 +4,399 @@ All notable changes to Router will be documented in this file.
+telemetry:
+  exporters:
+    metrics:
+      common:
+        service_name: apollo-router
+        views:
+          - name: apollo_router_http_request_duration_seconds # Instrument name you want to edit. You can use wildcard in names. If you want to target all instruments just use '*'
+            aggregation: drop
+
+```
+
+By [@bnjjj](https://github.com/bnjjj) in https://github.com/apollographql/router/pull/5531
+
+### Add `operation_name` selector for router service in custom telemetry ([PR #5392](https://github.com/apollographql/router/pull/5392))
+
+Adds an `operation_name` selector for the [router service](https://www.apollographql.com/docs/router/configuration/telemetry/instrumentation/selectors#router).
+Previously, accessing `operation_name` was only possible through the `response_context` router service selector.
+
+For example:
+
+```yaml
+telemetry:
+  instrumentation:
+    instruments:
+      router:
+        http.server.request.duration:
+          attributes:
+            graphql.operation.name:
+              operation_name: string
+```
+
+By [@bnjjj](https://github.com/bnjjj) in https://github.com/apollographql/router/pull/5392
+
+## 🐛 Fixes
+
+### Fix Cache-Control aggregation and age calculation in entity caching ([PR #5463](https://github.com/apollographql/router/pull/5463))
+
+Enhances the reliability of caching behaviors in the entity cache feature by:
+
+- Ensuring the proper calculation of `max-age` and `s-max-age` fields in the `Cache-Control` header sent to clients.
+- Setting appropriate default values if a subgraph does not provide a `Cache-Control` header.
+- Guaranteeing that the `Cache-Control` header is aggregated consistently, even if the plugins is disabled entirely or on specific subgraphs.
+
+By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/pull/5463
+
+### Fix telemetry events when trace isn't sampled and preserve attribute types ([PR #5464](https://github.com/apollographql/router/pull/5464))
+
+Improves accuracy and performance of event telemetry by:
+
+- Displaying custom event attributes even if the trace is not sampled 
+- Preserving original attribute type instead of converting it to string
+- Ensuring `http.response.body.size` and `http.request.body.size` attributes are treated as numbers, not strings
+
+> :warning: Exercise caution if you have monitoring enabled on your logs, as attribute types may have changed. For example, attributes like `http.response.status_code` are now numbers (`200`) instead of strings (`"200"`). 
+
+
+By [@bnjjj](https://github.com/bnjjj) in https://github.com/apollographql/router/pull/5464
+
+### Update the query planner to Federation 2.8.1 ([PR #5483](https://github.com/apollographql/router/pull/5483))
+
+Updates the query planner to v2.8.1 to [fix a performance issue when looking up contexts](https://github.com/apollographql/federation/pull/3031). 
+
+By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/pull/5483
+
```

Comment:
This changelog shoudl have been removed.  This version bump already released in https://github.com/apollographql/router/releases/tag/v1.49.1 https://github.com/apollographql/router/commit/6180312a55f3e2469c175b4813e02e4038204dcb

```suggestion
```